#Give the Object
string object_name
#Request is an integer called “start”
int32 start
---
#Response is a boolean called “confirm”
bool confirm
string message
bool arrived  # NEW: true if robot has reached destination
